dev/tools/controllerbuilder/pkg/codegen/mappergenerator.go

```diff
@@ -191,7 +191,7 @@ func (v *MapperGenerator) GenerateMappers() error {
 			// TODO: Can we figure out a better way here?
 			switch pbPackage {
 			case "cloud.google.com/go/bigtable/admin/apiv2/adminpb":
-				pbPackage = "google.golang.org/genproto/googleapis/bigtable/admin/v2"
+				// pbPackage = "google.golang.org/genproto/googleapis/bigtable/admin/v2"
```

Comment:
This is an deprecated library, used by [pkg/controller/direct/bigtable/instance_mapper.go](https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/8f39ac58feecdcca15a3265bf07c4e8377cb8de3/pkg/controller/direct/bigtable/instance_mapper.go#L24) and [pkg/controller/direct/bigtable/authorizedview_mapper.go](https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/8f39ac58feecdcca15a3265bf07c4e8377cb8de3/pkg/controller/direct/bigtable/authorizedview_mapper.go#L20). But the generated mapper file uses "cloud.google.com/go/bigtable/admin/apiv2/adminpb".

Reply:
Do you mean KCC never hits this line of code?

Reply:
It does, and the code will change the import library in `mapper.generated.go` to the deprecated `google.golang.org/genproto/googleapis/bigtable/admin/v2`, which is why this line needs to be removed.